hw/ip/sram_ctrl/data/sram_ctrl.hjson

```diff
@@ -17,7 +17,7 @@
   dv_doc:             "../doc/dv",
   hw_checklist:       "../doc/checklist",
   sw_checklist:       "/sw/device/lib/dif/dif_sram_ctrl",
-  version:            "1.1.0",
+  version:            "1.2.0",
```

Comment:
Increasing minor version because _readback_ functionality got added in a backwards-compatible way for what concerns SW

Reply:
Good catch, thanks! Fixed